### DISCUSSION TITLE
Texture not showing in Spoke when using Background Node and Lightmap - Only Black

### DISCUSSION THREAD

**Discussion by simonsais** [mozilla/hubs#4246] [Spoke Questions] [answered]
i followed the tutorial on making separate baked light maps, and without using the texture itself, the lightmap works. but when i add the texture via background node, the object just shows black. could somebody help?

![grafik](https://user-images.githubusercontent.com/83207539/117440808-d7495680-af34-11eb-89eb-4e8c720fb295.png)

![grafik](https://user-images.githubusercontent.com/83207539/117440702-b1bc4d00-af34-11eb-9f54-6a83a2abdaa4.png)

**Comment by j-conrad** [comment]
A couple things to try:
1. Make sure that your image texture is SAVED before exporting. You must explicitly save the file somewhere.
2. Your graph doesn't show any UV Map nodes. You'll need to specify which UVs are assigned to the main material textures vs. the lightmap texture.

**Comment by simonsais** [comment]
thank you!
unfortunately it didnt work

![grafik](https://user-images.githubusercontent.com/83207539/117543161-f2db5c80-b01b-11eb-968f-0edbd871cfff.png)

![grafik](https://user-images.githubusercontent.com/83207539/117543292-739a5880-b01c-11eb-844f-fdf16ae1c478.png)

**Comment by simonsais** [comment, marked as answer]
found the solution! 

i have to make a separate lightmapUV 

like seen here:
https://www.youtube.com/watch?v=0jegpQEkNX0

**Reply by Softvision-GeluHaiduc** [reply]
Marking this as answered, unmark the answer if you need any more information. Thank you!
